packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.test.ts

```diff
@@ -38,15 +38,13 @@ describe('Adapter is created as expected', () => {
     const locator = { meetingId: '123', passcode: 'qwe' };
     const statefulChatClient = createStatefulChatClientMock(new StubChatThreadClient());
     const threadClient = statefulChatClient.getChatThreadClient('threadId');
-    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
Bumping this comment as old comments can get lost on the PR thread https://github.com/Azure/communication-ui-library/pull/4344/files#r1546523458